packages/mdast-util-split/src/index.js

```diff
@@ -0,0 +1,124 @@
+module.exports = split
+import visit from 'unist-util-visit'
+
+function split (tree, {splitDepth = 1,
+  introductionAsProperty = true, conclusionAsProperty = false}) {
+  if (splitDepth < 1) {
```

Comment:
Isn't this assertion redundant with the default value set in the constructor?

Reply:
one could try to passe a negative integer. I don't know if in JS we make assertions when the boundaries are documented.

Reply:
I wouldn't since passing `true` or `'foo'` wouldn't throw here, yet they aren't positive integers >= 1.

Reply:
Removed.